composer.json

```diff
@@ -28,7 +28,8 @@
         "php-console/php-console": "^3.1.3",
         "jakub-onderka/php-parallel-lint": "^0.9",
         "predis/predis": "^1.1",
-        "phpspec/prophecy": "^1.6.1"
+        "phpspec/prophecy": "^1.6.1",
+        "rollbar/rollbar": "^1.3"
```

Comment:
i am not sure this should be only 2 digits, what if there is stuff that goes on 1.4 or beyond?

Reply:
I don't think I understand? We do want to allow non-breaking updates so I'm allowing everything up to `2.0.0`. Am I missing something?

Reply:
maybe 1.5.x will have something that is somewhat breaking, or additional things not taken into account here. Hinting that this leverages those additional things is too presumptuous i guess.

Reply:
We follow the semantic versioning standard so we do not introduce any backward incompatible changes in `1.*`. If there is anything backward incompatible that we need to introduce, we bump the major to `2.*.*`. I believe `^1.3` constraint supports that the best. What do you think?

Reply:
@ArturMoczulski oh i see you are from rollbar itself, pardon my presumptuousness 👴

Reply:
👍

Reply:
No worries. I appreciate your input ❤️️